src/main/java/com/aws/greengrass/lifecyclemanager/KernelLifecycle.java

```diff
@@ -366,6 +372,18 @@ void initConfigAndTlog() {
         } catch (IOException t) {
             logger.atError().log("Error finding built in service plugins", t);
         }
+
+        for (Class<? extends GreengrassService> cl : Arrays
+                .asList(DockerApplicationManagerService.class, UpdateSystemPolicyService.class,
+                        DeploymentService.class, FleetStatusService.class, TelemetryAgent.class,
+                        TokenExchangeService.class)) {
```

Comment:
nit: make this a constant like `Class<? extends GreengrassService> BUILTIN_SERVICE = Arrays.asList(....)` so it's more obvious to add to it if we ever add a new service?